src/cpp/src/storage/checkpointer.cpp

```diff
@@ -33,88 +61,87 @@ void Checkpointer::save(string checkpoint_dir, CheckpointMeta checkpoint_meta) {
-//                                           + PathConstants::relation_mapping_file;
-//
-//        string output_relation_mapping_filename = checkpoint_dir
-//                                                  + PathConstants::edges_directory
-//                                                  + PathConstants::relation_mapping_file;
-//
-//        if (fileExists(relation_mapping_filename)) {
-//            copyFile(relation_mapping_filename, output_relation_mapping_filename);
-//        }
-//
-//        // get node embeddings and model
-//        if (checkpoint_meta.has_model) {
-//            string node_embedding_filename = storage_->base_directory_
-//                                             + PathConstants::nodes_directory
-//                                             + PathConstants::embeddings_file
-//                                             + PathConstants::file_ext;
-//
-//            string output_node_embedding_filename = checkpoint_dir
-//                                                    + PathConstants::nodes_directory
-//                                                    + PathConstants::embeddings_file
-//                                                    + PathConstants::file_ext;
-//
-//
-//            if (fileExists(node_embedding_filename)) {
-//                copyFile(node_embedding_filename, output_node_embedding_filename);
-//            }
-//
-//            string model_filename = storage_->base_directory_ + PathConstants::model_file;
-//            string output_model_filename = checkpoint_dir + PathConstants::model_file;
-//
-//            if (fileExists(model_filename)) {
-//                copyFile(model_filename, output_model_filename);
-//            }
-//
-//            if (checkpoint_meta.has_state) {
-//                string node_state_filename = storage_->base_directory_
-//                                             + PathConstants::nodes_directory
-//                                             + PathConstants::embeddings_state_file
-//                                             + PathConstants::file_ext;
-//
-//                string output_node_state_filename = checkpoint_dir
-//                                                    + PathConstants::nodes_directory
-//                                                    + PathConstants::embeddings_state_file
-//                                                    + PathConstants::file_ext;
-//
-//                if (fileExists(node_state_filename)) {
-//                    copyFile(node_state_filename, output_node_state_filename);
-//                }
-//            }
-//        }
-//        copyFile(storage_->base_directory_ + PathConstants::config_file, checkpoint_dir + PathConstants::config_file);
-//
-//        copyFile(storage_->base_directory_ + PathConstants::checkpoint_metadata_file, checkpoint_dir + PathConstants::checkpoint_metadata_file);
-//    }
+    //    if (storage_->base_directory_ != checkpoint_dir) {
```

Comment:
auto-formatter changes. Also, should we remove this commented code?

Reply:
Yep this code is safe to delete